code/modules/antagonists/cult/runes.dm

```diff
@@ -57,11 +59,21 @@ Runes can either be invoked by one's self or with many different cultists. Each
 
 /obj/effect/rune/attackby(obj/I, mob/user, params)
 	if(istype(I, /obj/item/melee/cultblade/dagger) && iscultist(user))
+		if(log_when_erased)
+			var/confirm = alert(user, "Erasing this [cultist_name] rune might be against your goal to summon Nar'Sie.", "Begin to erase the [cultist_name] rune?", "Proceed", "Abort")
```

Comment:
input stalling is possible with this, if you for some reason want to erase your own summoning rune as a cultist without having a knife in your active hand

Reply:
Fair point. It's not realistic, but you should check that the user can still reasonably do this after the input is finished (has dagger in active hand, is cultist, and is alive/not incapacitated/whatever).

Reply:
I'd just ban you twice as hard for input stalling to grief your team anyway.

Reply:
Yeah, but this is still something that should be solved through code.